### PR TITLE
clear saved view proposal

### PR DESCRIPTION
**Pull request by manivoxel51**
## What changes are proposed in this pull request?

Proposal: Easier and more intuitive way to clear saved view by user. share your thoughts!

https://user-images.githubusercontent.com/109545780/218877666-cc463e2c-549b-43f1-a1d7-32b80b753489.mov

## How is this patch tested? If it is not, please explain why.

(Details)

### Is this a user-facing change that should be mentioned in the release notes?

-   [ ] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

**Review by kaixi-wang — APPROVED** [review]
I like it!

File: app/packages/components/src/components/Selection/Selection.tsx [hunk 1]
```diff
@@ -164,6 +173,24 @@ export default function Selection(props: SelectionProps) {
         startDecorator={
           <ColoredDot color={selectedColor || DEFAULT_COLOR_OPTION.color} />
         }
+        {...(selectedId !== "1" &&
```
Comment (imanjra) [inline app/packages/components/src/components/Selection/Selection.tsx:176]: Suggestion (may ignore): `selectedId !== fos. DEFAULT_SELECTED.id`

File: app/packages/core/src/components/Sidebar/ViewSelection/index.tsx [hunk 2]
```diff
@@ -214,6 +214,10 @@ export default function ViewSelection() {
             setSelected(item);
             setView(item.viewStages, [], item.slug);
           }}
+          onClear={() => {
+            setSelected(fos.DEFAULT_SELECTED);
```
Comment (imanjra) [inline app/packages/core/src/components/Sidebar/ViewSelection/index.tsx:218]: Suggestion (may ignore): `useResetRecoilState` could also be handy here

**Review by imanjra — APPROVED** [review]
Looks good. Nice work @manivoxel51 💥 
Left two comments that are just enhancements and can totally be disregarded or done as a follow-up

File: app/packages/components/src/components/Selection/Selection.tsx (hunk 1 above)
Comment (manivoxel51) [reply app/packages/components/src/components/Selection/Selection.tsx]: good idea!

**Pull request merged**